<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>581</width>
    <height>347</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>网络主机信息</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnGetHostInfo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>获取主机名和ip地址</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>541</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkBox_OnlyIPv4">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>110</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>只显示IPv4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="lookupHostInfo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>查找域名地址</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>www.ptpress.com.cn</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>www.baidu.com</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="btn_Add">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>10</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>allAddresses()</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_Inter">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>70</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>allInterfaces()</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>581</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
